SwiftLeeds/Views/My Conference/TalkCell.swift

```diff
@@ -32,14 +34,38 @@ struct TalkCell: View {
 
             VStack(alignment: .leading, spacing: 8) {
                 if let speaker = speaker {
-                    VStack(alignment: .leading, spacing: 4) {
-                        Text(speaker)
-                            .font(.headline.weight(.medium))
+                    HStack {
+                        VStack(alignment: .leading, spacing: 4) {
```

Comment:
Have we done a voiceover pass on any of these? The newer views are all configured to behave nicely but it took a smidge of extra work to do so.

Could be worth copying the approach over from some of the components?

Reply:
I'll take a look at VoiceOver and see how it behaves.